### PR TITLE
add code of conduct and add it to README

### PR DESCRIPTION
**Pull request by erizocosmico**
@campoy should we put anything else in the README regarding the CoC or is it okay with just the link to the md?

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at miguel@erizocosmi.co. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (campoy) [inline CODE_OF_CONDUCT.md]: I wonder if you should use a different email here?
Maybe something like conduct@sourced.tech?

Comment (erizocosmico) [reply CODE_OF_CONDUCT.md]: Oh damn, copied the generated CoC from one of my projects and it did have my email lol. Do we have such email? I'll change it in the meantime to `hello@sourced.tech`

Comment (campoy) [reply CODE_OF_CONDUCT.md]: We have that email, please use conduct@sourced.tech
After that you should merge

**Comment by erizocosmico** [comment]
Fixed @campoy

**Pull request merged**